cmd/show.go

```diff
@@ -36,8 +33,7 @@ func showContainers(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	ctx := context.Background()
-	hnd, err := podman.NewHandle(ctx, cOpts.PodmanSocket, cmdutil.NewLogger(cOpts.Verbose))
+	hnd, _, err := cOpts.GetHandle()
```

Comment:
What about logging here? In this file, there are multiple usage of fmt.Printf. Do we want to keep it that way (same usage in cmd/ports.go)?

Reply:
good point, clarified in a comment